spec/models/account_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Account, type: :model do
+  it 'should not allow the uuid to be updated' do
+    account  = create :account
+    uuid     = account.uuid
+    new_uuid = '9166953e-e71f-44aa-ba39-a6fe973a177e'
+
+    account.update!(uuid: new_uuid)
```

Comment:
Do we expect the `!` to raise an error? Or does the readonly setting let if fail silently? I'm guessing it must since the tests pass.

Reply:
Yeah, we expect it to _not_ raise an error; however, if the test breaks, either later, or during its development, we wanted it to raise something useful.

Reply:
gocha